assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -60,7 +60,94 @@ describe( 'modules/analytics accounts', () => {
 	} );
 
 	describe( 'actions', () => {
+		describe( 'createAccount', () => {
+			it( 'creates an account and adds it to the store ', async () => {
+				const accountName = fixtures.createAccount.accountName;
+				const propertyName = fixtures.createAccount.propertyName;
+				const profileName = fixtures.createAccount.profileName;
+				const timezone = fixtures.createAccount.timezone;
 
+				fetch
+					.doMockIf(
+						/^\/google-site-kit\/v1\/modules\/analytics\/data\/create-account-ticket/
+					)
+					.mockResponse(
+						JSON.stringify( fixtures.createAccount ),
+						{ status: 200 }
+					);
+
+				registry.dispatch( STORE_NAME ).createAccount( { accountName, propertyName, profileName, timezone } );
+				await subscribeUntil( registry,
+					() => (
+						registry.select( STORE_NAME ).isDoingCreateAccount() !== true
+					),
+				);
+
+				// Ensure the proper body parameters were sent.
+				expect( JSON.parse( fetch.mock.calls[ 0 ][ 1 ].body ).data ).toMatchObject(
+					{ accountName, propertyName, profileName, timezone }
+				);
```

Comment:
See above comment, this test is missing assertions for the API response being handled as expected. We should be able to look at `document.location` here to ensure it is the expected Analytics TOS URL with the same account ticket ID that we put in the `createAccount` fixture.

Reply:
👍 Will add that, I wasn't sure document.location would be set properly.

Reply:
Hmm, jsdom does not support navigation directly and throws an error. This approach might work to enable testing:

https://github.com/jsdom/jsdom/issues/2112#issuecomment-361195043

Reply:
Let's try using `location.assign` then, that sounds good to me.